app/forms/date_form.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import calendar
+
+from wtforms import Form, FormField, SelectField, StringField
+from wtforms import validators
+
+from app.validation.validators import DateCheck, DateRangeCheck, DateRequired, MonthYearCheck
+
+logger = logging.getLogger(__name__)
+
+
+def get_date_form(answer=None, to_field_data=None, validate_range=False, error_messages=None):
```

Comment:
I've tried multiple approaches to getting this working including __new__, __call__ and __init__. To be clear, what's being passed in is not state (i.e the data that will populate a form), but the form structure.